Tools/wasm/README.md

```diff
@@ -0,0 +1,54 @@
+# Python WebAssembly (WASM) build
+
+This directory contains configuration and helpers to facilitate cross
+compile CPython to WebAssembly (WASM).
+
+## wasm32-emscripten build
+
+Cross compiling to wasm32-emscripten platform needs [Emscripten](https://emscripten.org/)
+tool chain and a build Python interpreter.
```

Comment:
```suggestion
tool chain and a built Python interpreter.
```

Reply:
`build` here is I believe a reference to the autotools/configure `--build` argument. (Same for line 11 below).

Reply:
That is correct.  In lack of a better term I introduced the term "build Python"/

Reply:
My brain just keeps reading "build" as a verb instead of an adjective. 😄